DepositeCalcTests/Pages/CalculatorPage.cs

```
@@ -0,0 +1,210 @@
+        {
+            var selectDayDropDown = DayDropDown;
+            var selectDayDropDownElement = new SelectElement(selectDayDropDown);
+            selectDayDropDownElement.SelectByText(day);
+        }
+
+        public void SelectMonth(string month)
+        {
+            var selectMonthDropDown = MonthDropDown;
+            var selectMonthDropDownElement = new SelectElement(selectMonthDropDown);
+            selectMonthDropDownElement.SelectByText(month);
+        }
+
+        public void SelectYear(string year)
+        {
+            var selectYearDropDown = YearDropDown;
+            var selectYearhDropDownElement = new SelectElement(selectYearDropDown);
+            selectYearhDropDownElement.SelectByText(year);
+        }
+
+        public string StartDateMonth
+        {
+            get
+            {
+                return MonthDropDown.GetAttribute("value");
+            }
+
+            set
+            {
+                if (value == "360")
+                {
+                    FinancialYear360Days.Click();
+
+                    // $"{day}/{month}/{yeat}"
+                    return;
+                }
+
+                if (value == "365")
+                {
+                    FinancialYear365Days.Click();
+                    return;
+                }
+
+                throw new Exception("Invalid financial year value");
+            }
+        }
+
+        public string GetMonthEndDateText()
+        {
+            string EndDate = EndDateFld.GetAttribute("value");
+            return EndDate.Substring(3, 2);
+        }
+
+        public string monthNumberParseToText()
+        {
+            string numberStr = GetMonthEndDateText();
+            int number;
+
+            bool isParsable = Int32.TryParse(numberStr, out number);
+
+            if (isParsable)
+                return CultureInfo.InvariantCulture.DateTimeFormat.GetMonthName(number);
+            else
+                return null;
```

If something wrong happens, it's better to throw an exception.

```suggestion
                thow new Exception($"Invalid month number: {number}");
```